### ISSUE TITLE
исключение node_modules

### ISSUE THREAD

**Issue by dnfrolov** [Yauheni-Butski/NodeChatYauheniButski#4]
Желательно исключение node_modules
Посмотреть в сторону git ignore

**Comment by Yauheni-Butski** [comment]
@dnfrolov  Да, я помню что на тренинге упоминали о том, что в больших проектах лучше папку node_modules не мержить. Но в данном случае, заливал уже целиком, боялся что могут возникнуть какие-нибудь трудности в работоспособности без них, поэтому перестраховался.
Но ок, замечание учту.
